RedirectApplication/RedirectApplication/Controllers/ValuesController.cs

```diff
@@ -55,4 +53,4 @@ public void Delete(int id)
         {
         }
     }
-}
+}
```

Comment:
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline